plugin.go

```diff
@@ -11,11 +12,32 @@ type Logger interface {
 }
 
 type Plugin struct {
-	log *zap.Logger
+	log    *zap.Logger
+	config *Config
 }
 
-func (p *Plugin) Init(log Logger) error {
+type Configurer interface {
+	// UnmarshalKey takes a single key and unmarshal it into a Struct.
+	UnmarshalKey(name string, out any) error
+	// Has checks if a config section exists.
+	Has(name string) bool
+	// RRVersion is the roadrunner current version
+	RRVersion() string
+}
+
+func (p *Plugin) Init(cfg Configurer, log Logger) error {
+	const (
+		op            = errors.Op("app_plugin_init")
+		configSection = "app-logger"
```

Comment:
Config section is a plugin name. It should be `pluginName = "app-logger"`

Reply:
plugin name could not be changed as far as it is breaking change, and it is used by RPC clients